lib/console.js

```diff
@@ -35,26 +35,25 @@ function Console(stdout, stderr) {
   }
 }
 
-Console.prototype.log = function() {
-  this._stdout.write(util.format.apply(null, arguments) + '\n');
+Console.prototype.log = function(...args) {
+  this._stdout.write(`${util.format(...args)}\n`);
```

Comment:
Isn't `...args` still slightly slower than `.apply`? 

Not that it would probably be measurable given `log`.